crypto/keyring/keyring.go

```diff
@@ -755,7 +755,7 @@ func newRealPrompt(dir string, buf io.Reader) func(string) (string, error) {
 				continue
 			}
 
-			if err := os.WriteFile(dir+"/keyhash", passwordHash, 0o555); err != nil {
+			if err := os.WriteFile(dir+"/keyhash", passwordHash, 0o600); err != nil {
```

Comment:
Why does this change?

Reply:
The linter complains. Please refer: https://github.com/Finschia/finschia-sdk/pull/1043/commits/c4ffcdcaf9b7639efe4030bf182f412776a1544a